<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>computerMove</class>
 <widget class="QDialog" name="computerMove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelComputer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Der Computer gibt an sein Ergebnis ist:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="computerResult">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBelieve">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Believe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLie">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Lie!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
